packages/lesswrong/components/posts/PostsItemDate.tsx

```diff
@@ -116,11 +123,13 @@ const PostsItemDate = ({post, noStyles, includeAgo, useCuratedDate, classes}: {
     </LWTooltip>
   }
 
+  const isEmphasized = emphasizeIfNew && moment().diff(post.postedAt, 'days') < 2;
```

Comment:
https://github.com/ForumMagnum/ForumMagnum/pull/9197 suggests that we should be using `useCurrentTime` in preference to `moment()`/`new Date()` for comparing to whatever the current timestamp is, in contexts where we might have an SSR mismatch.  (I haven't looked too deeply into that yet, but it seems at least plausible to me.)